action.yml

```diff
@@ -33,6 +33,10 @@ inputs:
     description: 'Bash script to be run after make test inside the build folder'
     required: false
     default: ''
+  gzdev_project_name:
+    description: 'Project name to in gzdev repository configuration. Empty not to use gzdev'
```

Comment:
Is it a problem if we use `gzdev` every time? Or are we just keeping this optional for backwards compatibility?

Reply:
> Is it a problem if we use `gzdev` every time?

No that I can see, a part from bugs in gzdev itself and a bit of overhead.

> Or are we just keeping this optional for backwards compatibility?

The later :) I assumed crrent code is working, let's keep it or even simplify the action for third parties. Not an strong position though, fine if you think would be better to do it unconditionally.

Reply:
> simplify the action for third parties.

This use case hadn't even crossed my mind, do you think it's worth the trouble of keeping this general? 

> fine if you think would be better to do it unconditionally.

I was mainly hoping not to need to add more configuration if we can avoid it.

Reply:
I just removed the option of not using gzdev in c404850